### ISSUE TITLE
Fix Request: no primitive formal parameter false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#189]
lastModifiedTime has a violation, false positive since it is not a primitive:

```
public final class TimeUtils {
    private TimeUtils(){}
    public static boolean isExpired(LocalDateTime lastModifiedTime, long expiryDurationInHours){ // good
        LocalDateTime now = LocalDateTime.now();
        Duration duration = Duration.between(lastModifiedTime, now);
        long durationInHours = duration.abs().toHours();
        return durationInHours > expiryDurationInHours;
    }
}
```
